pkg/reconciler/serverlessservice/serverlessservice.go

```diff
@@ -28,7 +28,6 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/equality"
 	"k8s.io/apimachinery/pkg/api/errors"
```

Comment:
I'd actually remove the other import, given that it may collide with built in `errors` package.
Same for the other file.